### PR TITLE
remove firebase and signup, login, profile, lti login support, other unused things

### PR DESCRIPTION
**Pull request by mlippert**
## Description
Grant Thornton is only going to access and login via a link from Qualtrics. Therefore this is no need to support firebase users, creating them (signup), using them (login) or displaying them (profile). In addition they will never be accessing this app via an LMS/LTI login path.

## Motivation and context
So all the code to support that is unneeded, and better if it isn't there possibly causing unexpected behavior.

## How has this been tested?
Tested locally on my dev machine.

## Types of changes

- ✅ Breaking change (fix or feature that would cause existing functionality to change) Removing functionality.

## Checklist:

- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ All new and existing tests passed. **[REQUIRED]**
- ✅ My change requires a change to the documentation and I have updated it accordingly.
- ✅ I have added tests to cover my changes.

### CODE REVIEW

File: src/redux/actions/auth.js [hunk 1]
```diff
@@ -14,29 +14,14 @@
...
-  CLEAR_ERROR,
   CREATE_USER_FAIL,
   CREATE_USER_SUCCESS,
-  DISMISS_RESET_PASSWORD,
-  FORGOT_PASSWORD,
   FORGOT_PASS_FAIL,
-  INPUT_STATE_CHANGE,
-  LOGIN_ANONYMOUS,
   LOGIN_USER_FAIL,
   LOGIN_USER_SUCCESS,
```
Comment (brecriffs) [inline src/redux/actions/auth.js]: I am guessing that the actions left here, will be used by the Qualtrics login or something?

Comment (mlippert) [reply src/redux/actions/auth.js]: Actually probably not. I just removed the easy stuff that eslint told me about. You are correct that there is a more thorough clean up to be done if we want.

File: src/components/App.jsx [hunk 2]
```diff
@@ -209,41 +180,6 @@ class App extends React.Component {
...
-    }
-    else if (!this.props.auth.user.uid && !this.props.auth.uid) {
-      logger.debug('App.WillMount: No user (named or anonymous) detected');
-    }
-  }
-
-  componentDidMount() {
-    logger.debug('App: component loaded.');
-  }
-
```
[the remark this answers is not in the document]
Comment (mlippert) [reply src/components/App.jsx]: I couldn't think of any reason I'd want that log message, so I removed it.

**Comment by mlippert** [comment]
> I am unsure as to what query params are expected. Will not having the query params affect local development?

You can see the query params used in [loginFromQueryParams](https://github.com/rifflearning/riff-rtc/blob/gt-develop/src/components/App.jsx#L191-L210), they are `UUID`, `DisplayName`, `Room`.
I've been using this for local testing:
`https://localhost/room?UUID=12345&DisplayName=Me&Room=it-works` or this was even better for me:
`https://localhost/?UUID=V4Kc1uN0pgP7oVhaDjcmp6swV2F3&DisplayName=Me&Room=it-works` because that is my usual ID and my local riffdata has meetings for that user.

I have noticed some issues that need to be addressed, but they aren't applicable to this PR.

In particular refreshing the page w/o the query params, probably should work, and it does for the uid, but not for the display name and room name. The `roomName` and `displayName` probably just need to be removed from the chat persist blacklist.

File: src/components/App.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (brecriffs) [reply src/components/App.jsx]: Yeah it doesn't really provide anything useful I guess.

File: src/redux/actions/auth.js (hunk 1 above)
Comment (brecriffs) [reply src/redux/actions/auth.js]: Oh okay, makes sense.

**Comment by brecriffs** [comment]
> > I am unsure as to what query params are expected. Will not having the query params affect local development?
> 
> You can see the query params used in [loginFromQueryParams](https://github.com/rifflearning/riff-rtc/blob/gt-develop/src/components/App.jsx#L191-L210), they are `UUID`, `DisplayName`, `Room`.
> I've been using this for local testing:
> `https://localhost/room?UUID=12345&DisplayName=Me&Room=it-works` or this was even better for me:
> `https://localhost/?UUID=V4Kc1uN0pgP7oVhaDjcmp6swV2F3&DisplayName=Me&Room=it-works` because that is my usual ID and my local riffdata has meetings for that user.
> 
> I have noticed some issues that need to be addressed, but they aren't applicable to this PR.
> 
> In particular refreshing the page w/o the query params, probably should work, and it does for the uid, but not for the display name and room name. The `roomName` and `displayName` probably just need to be removed from the chat persist blacklist.

Ok thanks. I could have looked into more myself. And I think this question would have been more relevant to John's PR which initially set this up.

**Comment by mlippert** [comment]
@brecriffs Those were all reasonable and good review comments, thank you for reviewing.

**Comment by mlippert** [comment]
**Release** [gt-1.0.0-dev.3](https://github.com/rifflearning/riff-rtc/releases/tag/gt-1.0.0-dev.3)

**Pull request merged**